lib/kernel/src/code_server.erl

```diff
@@ -1163,7 +1175,41 @@ load_file_1(Mod, From, St) ->
 	error ->
 	    {reply,{error,nofile},St};
 	{Mod,Binary,File} ->
-	    try_load_module_1(File, Mod, Binary, From, St)
+            case is_loaded(features, St#state.moddb) of
+                false ->
+                    Allowed = true,
+                    persistent_term:put(Mod, no_features);
```

Comment:
This use of `persistent_term` can clash with the module's own use of persistent terms, since the documentation has the following recommendation:

> It is recommended to use keys like `?MODULE` or `{?MODULE,SubKey}` to avoid name collisions.

I suggest using an ETS table instead. (Not sure whether the code_sever should own it or if there should be a `features` process.)

Reply:
The use of `persistent_term` is just old debug code, which will be removed.